website/pages/docs/configuration/service-registration/consul.mdx

```diff
@@ -17,6 +17,8 @@ a default health check.
 - **HashiCorp Supported** – Consul Service Registration is officially supported
   by HashiCorp.
 
+~> **Version information** Service Registration is only available on Vault 1.4.0 and above.
```

Comment:
Hi! Thanks for working on this! Seems like it could be very helpful.

This is partially true. The `service_registration` stanza was explicitly added in 1.4.0; _however_, before it existed, Vault still registered itself as a service in Consul if Consul was configured as its physical back-end. Adding the stanza made it so that even if you're using Consul for storage, you could still have Vault advertise itself elsewhere, such as in Kubernetes.

Would it make sense to word this slightly differently to indicate that it was only the `service_registration` parameter that was added in 1.4.0?

Reply:
I have improved the language to clarify that consul automatically registers vault as a service when using the consul physical backend. 

Hashicorp documentation has a consistent "tone" that I want to preserve. If there is a better way to phrase the sentences I will gladly adjust per your recommendation.